zerver/models.py

```diff
@@ -1967,19 +1967,21 @@ class UserHotspot(models.Model):
     class Meta:
         unique_together = ("user", "hotspot")
 
-def check_valid_user_id(realm_id: int, user_id: object,
-                        allow_deactivated: bool=False) -> Optional[str]:
-    if not isinstance(user_id, int):
-        return _("User id is not an integer")
+def check_valid_user_ids(realm_id: int, user_ids: List[int],
+                         allow_deactivated: bool=False) -> Optional[str]:
+    error = check_list(check_int)("User IDs", user_ids)
+    if error:
+        return error
     try:
```

Comment:
It'd be better to have the `try/except` around the user fetch only, not the whole block.